OpenAIChatGPTBlazor/Pages/Index.razor

```diff
@@ -69,20 +68,20 @@
         }
     </div>
     <hr />
-    <div class="row">
+    <div class="row footer">
```

Comment:
this could now use a slight margin (or padding?) for the bottom for full screen view. Not sure how to achieve this the best way